portfolio/src/main/webapp/index.html

```diff
@@ -5,6 +5,7 @@
   <title>My Portfolio</title>
   <link rel="stylesheet" href="style.css">
 </head>
+<!-- prettier-ignore -->
```

Comment:
which area is being ignored and why?

Reply:
I was going to use prettier and didn't want it to format the HTML because I don't really like how it looks, but I'm actually not using prettier yet. Removed.